src/pages/en/guides/images.md

```diff
@@ -0,0 +1,228 @@
+
+```mdx
+// src/pages/post-1.md
+
+import rocket from '..images/rocket.svg';
+
+# My MDX Page
+
+// Local image stored at src/images/rocket.svg
+<img src={rocket} alt="A rocketship in space."/>
+
+// Local image stored at public/assets/stars.png
+![A starry night sky.](/assets/stars.png)
+<img src="/assets/stars.png" alt="A starry night sky." />
+
+// Remote image on another server
+![Astro](https://astro.build/assets/logo.png)
+<img src="https://astro.build/assets/logo.png" width="25" alt="Astro" />
+```
+
+### In UI Framework Components
+
+When adding images in a [UI framework component](/en/core-concepts/framework-components/) (e.g React, Svelte), use the image syntax appropriate for that particular component's framework.
+
+## Where to keep images
+
+### `src/`
+Your images kept in `src/` can be imported and used by other components, and will be processed during Astro's build process. These images are imported from a **relative [file path](https://www.codecademy.com/resources/docs/html/file-paths)** and the image's `src` attribute should refer to the import. 
+
+
+```astro
+---
+// src/pages/index.astro
+
+// Access images in `src/images/`
+import logo from '../images/logo.png';
+---
+<img src={logo} width="40" alt="Astro" />
+```
+
+### `public/`
+
+The [`public/` directory](/en/core-concepts/project-structure/#public) is for files and assets that do not need to be processed during Astro’s build process. Images stored here will be copied into the build folder untouched. These are not imported into your `.astro` file, and the image's `src` attribute is **relative to the public folder**.
+
+```astro
+---
+// src/pages/index.astro
+
+// Access images in `public/images/`
+---
+<img src="/images/logo.png" />
+```
+
+## Astro's Image Integration
+
+:::caution
+When you install the `@astrojs/image` integration, `.astro` files will no longer be able to use standard HTML `<img>` tags for images located in your project `src`. All local images must use the integration components instead.
+
+See the [image integration guide](/en/guides/integrations-guide/image/) for more notes about this new, experimental feature!
+:::
+
+Astro's official image integration provides two different Astro components for rendering optimized images: `<Image />` and `<Picture />`.
+
+After [installing the integration](/en/guides/integrations-guide/image/#installation), you can import and use these two components wherever you can use Astro components, including `.mdx` files!
+
+:::note
+Astro's `<Image />` and `<Picture />` components cannot be used with images in your `public/` folder. Use standard HTML or Markdown image syntax instead.
+:::
+
+### `<Image />`
+
+Astro's [`<Image />` component](/en/guides/integrations-guide/image/#image-) allows you to optimize a single image and specify width, height and/or aspect ratio. You can even transform your image to a particular output format, which can be used to avoid checking the file type of remote images. 
```

Comment:
Not sure because of `and/or`, but maybe we are missing a serial comma here?
```suggestion
Astro's [`<Image />` component](/en/guides/integrations-guide/image/#image-) allows you to optimize a single image and specify width, height, and/or aspect ratio. You can even transform your image to a particular output format, which can be used to avoid checking the file type of remote images. 
```